src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -239,6 +239,21 @@ private function addFormSection(ArrayNodeDefinition $rootNode, callable $enableI
                         ->end()
                         // to be deprecated in Symfony 6.1
                         ->booleanNode('legacy_error_messages')->end()
+                        // to be deprecated in Symfony 7.1
+                        ->booleanNode('controllers_render_form_views')
+                            ->info('Enables "AbstractController::render()" method to convert FormInterface to FormView and setting 422 status code when invalid.')
+                            ->defaultFalse()
+                            ->validate()
+                                ->always()
+                                ->then(function ($v) {
+                                    if (false === $v) {
+                                        trigger_deprecation('symfony/framework-bundle', '6.2', 'Setting the "framework.form.controllers_render_form_views" option to "false" is deprecated. It will have no effect as of Symfony 7.0.');
```

Comment:
The message should probably be adapted a bit, because it will be confusing when it is not set at all (which will be the case for all existing projects)

Reply:
You're right, that was a quick copy/past from a previous similar option. Updated, thanks!